resteasy-spring-feature-pack/common/src/main/resources/layers/standalone/resteasy-spring/layer-spec.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" ?>
+<layer-spec xmlns="urn:jboss:galleon:layer-spec:2.0" name="resteasy-spring">
+    <props>
+        <prop name="org.wildfly.rule.xml-path" value="/WEB-INF/web.xml,/web-app/listener/listener-class,org.jboss.resteasy.plugins.server.servlet.ResteasyBootstrap"/>
```

Comment:
What does this property mean?

Reply:
@jamezp , the property means that if a web.xml file containing the xml element /web-app/listener/listener-class with value org.jboss.resteasy.plugins.server.servlet.ResteasyBootstrap is present, then the layer resteasy-spring is provisioned. Perhpas we need more rules for detection. This one seemed a mandatory rule.

Reply:
Ah, okay. I just wasn't sure what it meant is all. I don't know enough about Spring to know if that is required or not. There's nothing wrong with it, I just didn't know what it meant :)